generators/server/templates/src/main/java/package/config/_Constants.java

```diff
@@ -8,6 +8,7 @@
     //Regex for acceptable logins
     public static final String LOGIN_REGEX = "^[_'.@A-Za-z0-9-]*$";
     // Spring profile for development and production, see http://jhipster.github.io/profiles/
+    public static final String SPRING_PROFILE_TEST = "test";
```

Comment:
I mean putting "0" directly in the `application.yml` file should already work - I don't understand the issue